lightkurve/lightcurvefile.py

```diff
@@ -397,6 +405,20 @@ def __repr__(self):
         return('TessLightCurveFile(TICID: {})'.format(self.targetid))
 
     def get_lightcurve(self, flux_type, centroid_type='MOM_CENTR'):
+        if centroid_type+"1" in self.hdu[1].data.columns.names:
+            centroid_col = self.hdu[1].data[centroid_type + "1"][self.quality_mask]
+            centroid_row = self.hdu[1].data[centroid_type + "2"][self.quality_mask]
+        else:
+            centroid_col = self.hdu[1].data["TIME"][self.quality_mask]*0.0
+            centroid_row = self.hdu[1].data["TIME"][self.quality_mask]*0.0
+
+        if 'QUALITY' in self.hdu[1].data.columns.names:
+            quality_vector = self.hdu[1].data['QUALITY']
+        elif 'SAP_QUALITY' in self.hdu[1].data.columns.names:
+            quality_vector = self.hdu[1].data['SAP_QUALITY']
+        else:
+            quality_vector = self.hdu[1].data['TIME']*0.0
```

Comment:
I recommend assigning `self._quality_vector` in the constructor and re-using it here, so we avoid code duplication.